continuous_integration/gpuci/axis.yaml

```
@@ -1,6 +1,6 @@
 PYTHON_VER:
-- "3.8"
-- "3.10"
+- "3.9"
+- "3.11"
```

We're currently only publishing GPU CI images for python 3.9/3.10:

```suggestion
- "3.10"
```

Is there a specific need for images using python 3.11? If so can explore building them.

For reference, here are some relevant files for dask-image's GPU CI images:

- https://github.com/rapidsai/dask-build-environment/blob/main/ci/axis/dask.yaml - controls the matrix of images being built
- https://github.com/rapidsai/dask-build-environment/blob/main/dask_image/Dockerfile - controls the build steps of the images
- https://github.com/rapidsai/dask-build-environment/blob/main/dask_image/environment.yml - specifies the GPU-specific conda requirements to install in the CI environment

---

Went ahead and committed this change. We can revisit when there are Python 3.11 images